src/BirdLIST.js

```diff
@@ -0,0 +1,38 @@
+// import { useState } from "react";
+
+
+function BirdLIST({birdArray,birdCostArray}){
+  let birdiesrender=""
+        // const[birdarr, setbirdarr] = useState("")
+        // birdArray.forEach(element => {birdies+=`<li>${element}</li>`})
+        // birdCostArray.forEach(element => {cost+=`<li>${element}</li>`})
+
+        //         setbirdarr(birdies)
+for (let i=0;i<birdArray.length;i++){
```

Comment:
Could just do `birdArray.map(bird => (<li id=${bird.id}>${bird.name}: ${bird.price}</li>)` below in the render part